dbt/include/trino/macros/adapters.sql

```diff
@@ -151,11 +151,6 @@
 {% endmacro %}
 
 
-{% macro trino__get_batch_size() %}
```

Comment:
nit: Do the move of this method in a separate preparatory commit.
It does not really have anything to do with column types overrides.